--- a/py/mockbuild/package_manager.py
+++ b/py/mockbuild/package_manager.py
@@ -712,6 +712,8 @@
             for i, line in enumerate(e.msg.split('\n')):
                 if 'no such command: builddep' in line.lower():
                     raise BuildError("builddep command missing.\nPlease install package dnf-plugins-core.")
+                if 'is already installed' in line.lower():
+                    return
             raise
